### PR TITLE
Add toggle checkbox to testrig

### PR DESCRIPTION
**Pull request by maneeshnycda**
@BJK @bartriordan please review.

Per @bartriordan I have made it a checkbox toggle function that allows you to switch between `true` and `false` states within a test.

### CODE REVIEW

File: src/testUtils/TestRig.react.js [hunk 1]
```diff
@@ -54,6 +54,11 @@ export default class TestRig {
     var element = this.domNode.find(selector)
     this.fillInElement(element, value)
   }
+  toggleCheckbox(selector) {
+    var element = this.domNode.find(selector) // Selector should be specific to the checkbox
+    $(element).prop('checked', !($(element)[0].checked))
+  }
+
```
Comment (bartriordan) [inline src/testUtils/TestRig.react.js]: @maneeshnycda No need for this blank line.

File: src/testUtils/TestRig.react.js [hunk 2]
```diff
@@ -54,6 +54,11 @@ export default class TestRig {
     var element = this.domNode.find(selector)
     this.fillInElement(element, value)
   }
+  toggleCheckbox(selector) {
+    var element = this.domNode.find(selector) // Selector should be specific to the checkbox
```
Comment (bartriordan) [inline src/testUtils/TestRig.react.js]: @maneeshnycda How would you feel about changing this to:
```jsx
var element = $(this.domNode.find(selector))
```
?
That way the next line doesn't have to wrap `element` in `$()` twice--it's already done.

File: test/lib/specHelperSpec.js [hunk 3]
```diff
@@ -49,6 +49,7 @@ class TestComponent  extends React.Component {
         <h1>Test Component</h1>
         <div className='answer'>Form Value: {this.state.value}</div>
         <input ref='value' onChange={this.getVal.bind(this)} defaultValue={this.state.value} />
+        <input id='checkboxInput' type='checkbox' ref='checkValue' onChange={this.getVal.bind(this)} />
```
Comment (bartriordan) [inline test/lib/specHelperSpec.js]: @maneeshnycda Despite our preferences in React for camelCase, we generally use hyphens in CSS identifiers.

Comment (bartriordan) [inline test/lib/specHelperSpec.js]: @maneeshnycda `checkbox-input` is what I meant

**Comment by bartriordan** [comment]
@maneeshnycda Looks good. A few small comments.

File: src/testUtils/TestRig.react.js (hunk 2 above)
Comment (maneeshnycda) [inline src/testUtils/TestRig.react.js]: @bartriordan sure sounds reasonable.  I'll make the change.

Comment (bartriordan) [inline src/testUtils/TestRig.react.js]: @maneeshnycda Per our conversation this should be disregarded: the efficiency of wrapping `element` only once isn't worth breaking the pattern used in every other function.

File: test/lib/specHelperSpec.js (hunk 3 above)
Comment (maneeshnycda) [inline test/lib/specHelperSpec.js]: @bartriordan hmm okk

**Comment by bartriordan** [comment]
@maneeshnycda Merging.

**Pull request merged**